content/docs/querying/operators.md

```diff
@@ -195,7 +196,7 @@ If the metric `http_requests_total` had time series that fan out by
 `application`, `instance`, and `group` labels, we could calculate the total
 number of seen HTTP requests per application and group over all instances via:
 
-    sum(http_requests_total) by (application, group)
+    sum(http_requests_total) without (instance)
```

Comment:
Yeah, that's a risk. Thankfully labels change pretty rarely (I think we've had one in the node exporter, and I'm expecting maybe one more) whereas every user has a different label taxonomy.